.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: ruby
 cache: bundler
 rvm:
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-9.0.0.0
+  - jruby-9.0.4.0
```

Comment:
Seems to be super-slow with 9.0.4.0. For me it's ok if we just run with 9.0.0.0.